iconpacks/list.json

```
@@ -33,6 +33,17 @@
       "suffix": "",
       "config": "https://raw.githubusercontent.com/Panniku/vd-iconpacks/master/Packs/MaterialDesign3/config.json",
       "load": "https://raw.githubusercontent.com/Panniku/vd-iconpacks/master/Packs/MaterialDesign3/"
+    },
+    {
+      "id": "Solar",
+      "description": "Giant icon pack as big as Solar System!",
+      "credits": {
+        "authors": ["Moodle <807170846497570848>"],
+        "source": "https://www.figma.com/community/file/1166831539721848736/"
+      },
+      "suffix": "@2x",
+      "config": "https://raw.githubusercontent.com/nexpid/VendettaThemesPlus/main/iconpacks/configs/solar.json",
```

```suggestion
      "config": "https://raw.githubusercontent.com/Moodzz1/discord-iconpacks/master/Packs/Solar/solar.json",
```
ditto, copy the config.json file over to your repo